mlrun/api/api/endpoints/projects.py

```diff
@@ -322,6 +324,100 @@ async def get_project_summary(
     return project_summary
 
 
+@router.post("/projects/{name}/load", response_model=mlrun.api.schemas.BackgroundTask)
+async def load_project(
+    name: str,
+    url: str,
+    background_tasks: fastapi.BackgroundTasks,
+    auth_info: mlrun.api.schemas.AuthInfo = fastapi.Depends(
+        mlrun.api.api.deps.authenticate_request
+    ),
+    db_session: sqlalchemy.orm.Session = fastapi.Depends(
+        mlrun.api.api.deps.get_db_session
+    ),
+):
+    """
+    Loading a project remotely from a given source.
+
+    :param name:                project name
+    :param url:                 git or tar.gz or .zip sources archive path e.g.:
+                                git://github.com/mlrun/demo-xgb-project.git
+                                http://mysite/archived-project.zip
+                                The git project should include the project yaml file.
+    :param background_tasks:    injected automatically by fastapi
+    :param auth_info:           auth info of the request
+    :param db_session:          session that manages the current dialog with the database
+
+    :returns: a BackgroundTask object, with details on execution process and its status
+    """
+
+    def _run_load_project(
```

Comment:
I did it because both `create_background_task` and `mlrun.api.crud.WorkflowRunners().run` has project as a parameter... but I can overcome it in a more elegant way :)